src/app/(app)/shopping-cart.tsx

```diff
@@ -1,35 +1,72 @@
-import { FlashList } from '@shopify/flash-list';
-import { useCallback } from 'react';
+import { FlatList } from 'react-native';
 
-import type { Post } from '@/api';
-import { usePosts } from '@/api';
-import { Card } from '@/components/card';
-import { EmptyList, FocusAwareStatusBar, Text, View } from '@/ui';
+import { useShoppingCart } from '@/api/cart/use-line-items';
+import { HeaderLogo } from '@/components/header-logo';
+import { ShoppingCard } from '@/components/shopping-cart/shopping-card';
+import { Button, SafeAreaView, Text, View } from '@/ui';
 
-export default function Feed() {
-  const { data, isPending, isError } = usePosts();
-  const renderItem = useCallback(
-    ({ item }: { item: Post }) => <Card {...item} />,
-    []
-  );
+export default function ShoppingCart() {
+  const { data, hasNextPage, fetchNextPage, isFetchingNextPage, refetch } =
+    useShoppingCart({
+      variables: { items: 7 },
```

Comment:
NIT Pick. 
Consider moving this value to a const, the name of the variable should describe what this value is or do.